corehq/apps/hqcase/management/commands/ptop_fast_reindex_cases.py

```diff
@@ -7,5 +7,5 @@ class Command(ElasticReindexer):
     help = "Fast reindex of case elastic index by using the case view and reindexing cases"
 
     doc_class = CommCareCase
-    view_name = 'case/by_owner'
+    view_name = 'hqcase/by_owner'
```

Comment:
No, it worked before too. The two views just have the same salient property of emitting each case exactly once, so they can be used in the same way here.